luxtronik/datatypes.py

```diff
@@ -333,10 +334,45 @@ class Version(Base):
 
     datatype_class = "version"
 
+    def __init__(self, name, writeable=False):
+        """Initialize the version data field class. No children yet."""
+        Base.__init__(self, name, writeable)
+        self.children = []
+
+    @classmethod
+    def to_heatpump(cls, value):
+        """Converts value into heatpump units."""
+        raise NotImplementedError(
+            '"to_heatpump()" is not implemented for class "Version"'
+        )
+
     @classmethod
     def from_heatpump(cls, value):
         return "".join([chr(c) for c in value]).strip("\x00")
 
+    @Base.value.getter
```

Comment:
`pylint` complains here. Is this the correct way to overwrite the getter of `value` in the `Base` class?